### DISCUSSION TITLE
Can I use the SP-API for Amazon Advertising Reports?

### DISCUSSION THREAD

**Discussion by p96r** [saleweaver/python-amazon-sp-api#157] [FAQ]
Amazon provides Advertising API at [https://advertising.amazon.com/API/docs/en-us](https://advertising.amazon.com/API/docs/en-us), which is fully buggy and unpredictable. 

Is there any way we could use the SP-API for Advertising reports as well?

**Comment by saleweaver** [comment]
A client doesn't improve the servers performance. There are certain parts of the code you could use, but only because both APIs were created by amazon doesn't mean they have anything else in common. Amazon doesn't behave the same even on two endpoints within the same API

**Comment by ianrussel** [comment]
The answer is no.They have different parameters in request and etc
